Library/Homebrew/utils/json.rb

```diff
@@ -1,4 +1,4 @@
-require "vendor/okjson"
+require "json"
 
 module Utils
   module JSON
```

Comment:
@woodruffw We could consider removing this entire module and replacing it with calls to these methods instead.